types/ember__utils/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for @ember/utils 3.0
+// Project: http://emberjs.com/
+// Definitions by: Mike North <https://github.com/mike-north>
```

Comment:
We should figure out how we want this handled. Copy everything from '@types/ember' or start fresh?

Reply:
That's a great question. We have an ever-growing list of authors in `@types/ember` right now, which means an ever-growing list of people with the ability to sign off on changes going in. 

I'm not sure when folks add themselves to that list whether they're just viewing it as adding to a tally of contributors, or if they're intentionally setting themselves up to be pinged for future reviews, but I'm not sure how sustainable the current pattern is.